azure-pipelines.yml

```diff
@@ -51,7 +52,8 @@ jobs:
       projects: 'src/**/*.UnitTests.csproj'
       arguments: '--configuration $(buildConfiguration) --no-build'
 
-  - template: azure-pipelines-templates/dependency-check.yml@das-platform-building-blocks
+  - template: azure-pipelines-templates/build/step/app-build.yml@das-platform-building-blocks
```

Comment:
Would be good to make use of a pipeline-templates folder as seen in other apprentice-commitments repos